test/tools/unified-spec-runner/match.ts

```diff
@@ -467,7 +469,25 @@ function compareEvents(
       if (expectedEvent.poolClearedEvent.hasServiceId) {
         expect(actualEvent).property('serviceId').to.exist;
       }
-    } else if (validEmptyCmapEvent(expectedEvent, actualEvent)) {
+    } else if (validEmptyCmapEvent(expectedEvent as ExpectedCmapEvent, actualEvent as CmapEvent)) {
+      return;
+    } else if (expectedEvent.serverDescriptionChangedEvent) {
+      expect(actualEvent).to.be.instanceOf(ServerDescriptionChangedEvent);
+      const expectedServerDescriptionKeys = ['previousDescription', 'newDescription'];
+      expect(expectedServerDescriptionKeys).to.include.all.members(
+        Object.keys(expectedEvent.serverDescriptionChangedEvent)
+      );
```

Comment:
I'm trying to avoid sweeping changes in the runner, this just seemed like the right set of properties to check for the new event that's getting implemented